CONTRIBUTING.md

```diff
@@ -73,6 +74,75 @@ The current list of labels are [here](https://github.com/bids-standard/bids-spec
+Commenting on pull requests is an important way of participating in the BIDS community.
+
+Please follow the instructions for how to comment on a pull request.
+
+### Navigating to open pull requests
+
+Starting from our [BIDS-Specification GitHub](https://github.com/bids-standard/bids-specification), please click on `Pull requests`
+
+![BIDS-mainpage](commenting_images/BIDS_GitHub_mainpage.png "BIDS_GitHub_mainpage")
+
+### Selecting an open pull request
+
+In this example we will be navigating to our [BIDS common derivatives pull request](https://github.com/bids-standard/bids-specification/pull/265).
+
+![BIDS-pr-list](commenting_images/BIDS_pr_list.png "BIDS_pr_list")
+
+### Pull request description
+
+Upon opening the pull request we see a detailed description of what this pull request is seeking to address.
+Descriptions are important for reviewers and the community to gain context into what the pull request is achiving.
+
+![BIDS-pr](commenting_images/BIDS_pr.png "BIDS_pr")
+
+### Generally commenting on a pull request
+
+Please scroll to the bottom of the pull request to find a comment box.
+These comments will address the pull request in general.
+Next, we will go over how to comment on specific lines of the pull request.
+
+![BIDS-comment](commenting_images/BIDS_comment.png "BIDS-comment")
+
+### Specific comments on a pull request
+
+Please click on the `Files changed` tab from the pull request page.
+This tab displays what this pull request seeks to change.
+The green text is being added while the red is being removed.
+To comment on a specific line, please click on the `plus sign` (pictured below).
+To comment on several consecutive lines, please click and drag until the yellow highlight captures the lines of interest.
+
+![BIDS-specific-comment](commenting_images/BIDS_file_comment.png "BIDS-specific-comment")
+
+#### Suggesting text
+
+When you are making specific line comments, you may also suggest a text change.
+To do this, please first select the line(s) you would like to make a suggestion.
+Then please click the plus/minus icon in the comment box (pictured below).
+
+![BIDS-suggest-box](commenting_images/BIDS_suggest.png "BIDS-suggest")
+
+Once the button is clicked the highlighted text will be copied into the comment box and formatted as
+a [Markdown code block](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).
+
+![BIDS-suggest-text](commenting_images/BIDS_suggest_text.png "BIDS-suggest-box")
+
+As you work on the suggestion you may preview the comment by clicking the `Preview` button. 
+
+![BIDS-suggest-preview](commenting_images/BIDS_suggest_preview.png "BIDS-suggest-preview")
+
+When the text has changed, the preview will identify the differences.
+
+![BIDS-suggest-change](commenting_images/BIDS_suggest_change.png "BIDS-suggest-change")
```

Comment:
This is cutting it a bit fine, IMO. I think one image showing the preview and the highlighting gets the idea across plenty.

```suggestion
The "Preview" tab in the comment box will show your suggestion as it will be rendered.
The "Suggested change" box will highlight the differences between the original text and your suggestion.

![BIDS-suggest-change](commenting_images/BIDS_suggest_change.png "BIDS-suggest-change")
```